go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/dolmen-go/contextio v0.0.0-20200217195037-68fc5150bcd5
 	github.com/elastic/elastic-agent-autodiscover v0.9.0
 	github.com/elastic/elastic-agent-client/v7 v7.16.0
-	github.com/elastic/elastic-agent-libs v0.12.1
+	github.com/elastic/elastic-agent-libs v0.12.2-0.20241016085313-16569381887f
```

Comment:
once https://github.com/elastic/elastic-agent-libs/pull/240 is merged it will be updated to `v0.12.2`